examples/chip-tool/commands/pairing/PairingCommand.cpp

```diff
@@ -397,8 +397,8 @@ CHIP_ERROR PairingCommand::UpdateNetworkAddress()
 
 void PairingCommand::OnAddressUpdateComplete(NodeId nodeId, CHIP_ERROR err)
 {
-    ChipLogProgress(chipTool, "OnAddressUpdateComplete: %s", ErrorStr(err));
-    if (err != CHIP_NO_ERROR)
+    ChipLogProgress(chipTool, "OnAddressUpdateComplete: %" PRIx64 ": %s", nodeId, ErrorStr(err));
+    if (err != CHIP_NO_ERROR && nodeId == mRemoteId)
```

Comment:
Should it not be a PeerID here? Operational discovery is not just a Node ID, but a <Root PK, Fabric ID, Node ID> tuple, where we see <Root PK, Fabric ID> as compressed fabric ID over DNS-SD records. Right now, fabric is always 0, so that may explain NodeId alone, but is that something that can be cleaned-up in this PR, or is it too big a change?

Reply:
Yes, the operating assumption is that fabric is always zero, although some code uses 0x1b669 (I don't actually understand why some code uses 0 and other code uses 0x1b669, but given that there is not agreement about what the fabric id actually is we should not be doing equality comparisons with it).

Reply:
Well, there is clear normative guidance in spec, but overall, in this instance, for operational discovery during commissioning, root and fabric ID have to match between commissioner operational credentials (those used to establish CASE afterwards) and the root/fabric ID used for operational discovery. That's why I was confused PeerId was not used, and only NodeId.

Reply:
I believe @pan-apple is working on that.

I'd want to understand the reasons for the discrepancies that currently exist before shaving that yak.

Reply:
I am approving because this method signature is already problematic:

```
void PairingCommand::OnAddressUpdateComplete(NodeId nodeId, CHIP_ERROR err)
```
Because this PR isn't adding a new code path that ignores fabric, but is just adding to an existing code path that already ignores fabric, I don't feel that this PR is itself creating a problem.  It doesn't fix the problem, but isn't introducing it either.